### DISCUSSION TITLE
the property: background-image: url("/image.jpg"); is not working online, only in my local browser.

### DISCUSSION THREAD

**Discussion by jjpicturesahora** [community/community#59915] [Programming Help] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

the property: background-image: url("/image.jpg"); is not working online, only in my local browser. 

This is essential for the added value of my service as web developer and designer. It has appended in all my projects within Github. What can I do? Has any of it happened to anyone before?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by tshenolo** [comment]
Hi there

Try providing the absolute path of the image.
For example: background-image: url("/image.jpg") should be changed to 
background-image: url("http://example.com/image.jpg")

Note that url("/image.jpg") means that the image is in the top level folder of the web server and can be 
accessed via the browser as: http://example.com/image.jpg

Also note that you can change  background-image: url("/image.jpg") to  background-image: url("image.jpg")
if the index page and the image are in the same folder

**Reply by jjpicturesahora** [reply]
Thank you, but I am trying to use an image from my own repository rather than some stored elsewhere online. That is the issue. I have had no problem using an image from google and using it as a background. But what happens when I want you use a picture I have taken myself and it's stored in my own repository?

**Reply by GhostOf0days** [reply, marked as answer]
I would honestly, always add ./ for same level just for caution.

**Reply by jjpicturesahora** [reply]
Thank you my friend. Your answer was correct. I must add that the Live preview extension I was using showed to me as if just the slash "/", alone without the dot, was enough. Happily I applied your solution "./" and works everywhere. Applause!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @jjpicturesahora ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by GhostOf0days** [comment]
Which repo is it? Try adding ./ instead of /. Please look at this guide on [relative pathing](https://www.codingrooms.com/blog/file-paths) if this is the issue. Basically, ./ means start at the same directory as the file you are on (i.e. the file I'm trying to get is at the same level as the file I'm trying to get it in). ../ is saying to go one level back (I may be trying to get a file, but I'm in a folder, which is at the same level as the file). You can use multiple ../'s. Relative pathing is from the file you are on (or the page you are on if you are using relative pathing in JavaScript). Absolute paths start at the root, meaning they are relative to the very first level not the file you are on. Relative paths start with / just like you are doing.

**Reply by jjpicturesahora** [reply]
Thank you a lot, your answer was correct, "./" instead of  any other form. And thank you for the guide. Most useful!

**Comment by Beargruug** [comment]
you can move you're images into the public folder. after that you should be able to see them.

**Reply by jjpicturesahora** [reply]
Thank you for your comment. I had already done that, but thank you anyway.

**Comment by HuyNguyenChau** [comment]
You can use "./" and "../" 
You can use "../" to  the path will go back one directory and will work on github
